src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -309,3 +309,8 @@ export const HighlightConceptName = ({ conceptName, searchFilter }): ReactElemen
     span([conceptName.substring(endIndex)]),
   ]);
 };
+
+export const DisplayParticipantCount = (count: number): ReactElement => {
```

Comment:
```suggestion
export const DisplayParticipantCount = ({ count: number }): ReactElement => {
```

Reply:
If there is only argument, why does this need to be converted into an object?

Reply:
Great question!

It is because when this function is rendered as a FunctionalComponent, that argument becomes `props`. The object is to destructure `count` out of `props`.

The issue with not rendering it as a Component (and instead calling the function) is that you lose the react optimizations around rendering. (It will recreate the whole DOM tree every time)

You can think of `h(DisplayParticipantCount, { count: 20 }) as a wrapper around `React.createElement('DisplayParticipantCount', { count: 20 })` or `<DisplayParticipantCount count={20} />`. These all ultimately render `count` as a field on the component props.

Reply:
Great comment Sky!

Reply:
It doesn't apply anymore!